README.md

```diff
@@ -31,10 +31,12 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
+$ source /opt/ros/indigo/setup.bash
```

Comment:
I would change this to something like `source /path/to/my/workspace/setup.bash`. We can add a `# see below` comment and explain that you should source whatever workspace you are currently using. The default is `/opt/ros/indigo`, but @psigen pointed out that this is **not** correct if you've built a newer version of a dependency in a chained workspace.

This is not uncommon if you are stuck using an old version of ROS, but need to upgrade one package.